src/org/labkey/test/components/react/BaseReactSelect.java

```diff
@@ -477,7 +477,7 @@ private Locators()
     {
         private Locator.XPathLocator _locator;
         private boolean _mustBeEnabled = false;
-        private boolean _findParent = true;
+        private final boolean _findParent = true;
```

Comment:
Just remove this (and the `if` that references it).
The if is a no-op:
```
if (_findParent)
    tmpLoc.parent();
```

Reply:
OK, I'll make this change and push, then monitor results before merging